itests/qtest-iceberg/src/test/java/org/apache/hadoop/hive/cli/TestIcebergLlapLocalCompactorCliDriver.java

```diff
@@ -41,6 +41,8 @@ public class TestIcebergLlapLocalCompactorCliDriver {
 
   static CliAdapter adapter = new CliConfigs.IcebergLlapLocalCompactorCliConfig().getCliAdapter();
   private static final AtomicBoolean stop = new AtomicBoolean();
+  private static final String defaultPoolName = null;
+  private static final String icebergPoolName = "iceberg";
```

Comment:
minor: please check https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=5503&id=apache_hive&open=AZJ6FzB-hMQv6qrxfwyh

final vars should be all uppercase with underscore